Misc/reputation-registryKey.json

```diff
@@ -24,7 +24,7 @@
     "contextValue": "",
     "excludedBrands": [],
     "defaultMapping": {
-        "registrykeyname": {
+        "namefield": {
```

Comment:
is that on purpose?

Reply:
yeah, it's now pointing to `IndicatorFields/incidentfield-namefield.json`